src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1873,6 +1873,113 @@ private static bool TryGetCommandInfoList(PowerShell ps, HashSet<string> command
             return true;
         }
 
+        internal static OutputRendering OutputRenderingSetting = OutputRendering.Automatic;
+
+        internal static bool OutputIsPlainText(bool isHost, bool? supportsVirtualTerminal)
```

Comment:
When I read the name `OutputIsPlainText`, I over and over again think that it is something that tests the string to be outputted if it is plain text or not. Can we find another name that makes it clear that it is checking a setting?

Reply:
ShouldOutputPlainText?

Reply:
Will change, I like that better